torch/csrc/jit/codegen/cuda/scheduler/utils.cpp

```diff
@@ -1176,7 +1238,7 @@ IterDomain* innerMostRootDim(TensorView* tv) {
 FindAllMappedDims::FindAllMappedDims(
```

Comment:
Would it make sense to organize this pass as a propagation pass on a spanning tree?

Reply:
done